### PR TITLE
Shared calendars shouldn't allow adding attendees if the calendar is not yours

### LINKED ISSUE #5450: Shared calendars shouldn't allow adding attendees if the calendar is not yours
### Steps to reproduce

1. Open calendar app
2. Create and event and select a shared calendar
3. Add an attendee

### Expected behavior

I shouldnt be able to add an attendee in a calendar that is not mine because they will receive no invitation

### Actual behaviour

I can add an attendee, and thats confusing people the other person will not know that i sent them a request for a meeting

### Calendar app version

_No response_

### CalDAV-clients used

_No response_

### Browser

_No response_

### Client operating system

_No response_

### Server operating system

_No response_

### Web server

None

### Database engine version

None

### PHP engine version

None

### Nextcloud version

_No response_

### Updated from an older installed version or fresh install

None

### List of activated apps

_No response_

### Nextcloud configuration

_No response_

### Web server error log

_No response_

### Log file

_No response_

### Browser log

_No response_

### Additional info

_No response_

### PR DESCRIPTION
**Pull request by GretaD**
fixes #5450

Screenshot from 2023-09-07 09-16-12

### CODE REVIEW

**Review by ChristophWurst — APPROVED** [review]

File: src/components/Editor/Invitees/InviteesList.vue [hunk 1]
```diff
@@ -115,6 +119,9 @@ export default {
 		noInviteesMessage() {
 			return this.$t('calendar', 'No attendees yet')
 		},
+		noOwnerMessage() {
+			return this.$t('calendar', 'You dont own this calendar, so you cannot invite other people')
```
Comment (tcitworld) [inline src/components/Editor/Invitees/InviteesList.vue:123]: **suggestion:** 
```suggestion
			return this.$t('calendar', "You don't own this calendar, so you cannot add attendees to this event")
```
As attendees may not just be people in the future.

Comment (ChristophWurst) [reply src/components/Editor/Invitees/InviteesList.vue]: We should probably also disable rooms and resources

**Comment by nunoleite** [comment]
I'm sorry, but i disagree.

On our Nextcloud implementation we have a user that's the Admin, that manages some global things. Like folders and calendars. Because Nextcloud in centered on the User and not on Projects we have a user that created some resources to centralize Projects and that is shared to everyone.

So, what we have is a calendar that was created in the Admin user (no one uses this Admin).
We have a "Generic" calendar that is shared to everyone. Than we use this calendar to put events so everyone can know whats happening but these events are only for some people to attend.

We need to have a calendar to put events about a Project so everyone knows whats happening, but these events are related to only one project so only 2 or 3 persons will be attended.

And by the logic these 2 or 3 attendees should receive a notification by email, they only receive by Nextcloud App notification.
And if i have Edit permissions on this calendar why shouldn't i have permissions to edit the Attendees?
They should receive the email notification has it is on normal calendars.

Thanks

**Comment by ChristophWurst** [comment]
@nunoleite the problem is that this setup is broken. Nextcloud doesn't send out invites if you aren't the owner of the calendar. See https://github.com/nextcloud/server/issues/26668.

**Comment by outofcontrol** [comment]
@ChristophWurst  we have the patch in place for invites to go out which works very quite well, and our users have always managed our shared calendar. We just upgraded from v21 to v27 and find users can no longer add attenedees. I would ask that this change be reconsidered.

Ideally, and what IMO would make the calendar better, is to fix the invitation issue and not hobble the ability to maintain calendars. 

I believe this change to be taking NextCloud calendar in the wrong direction.

**Comment by kesselb** [comment]
> we have the patch in place for invites to go out which works very quite well

I advise against using https://github.com/nextcloud/server/pull/36756 or https://github.com/nextcloud/server/pull/39702. While invitations work, it still leaves many rough edges. For example, events are shown twice sometime. The behavior for create and update events is not aligned, and weird things happen when deleting such events.

**Comment by outofcontrol** [comment]
We've been using it for I think close to 2 years, seems fine. But that is neither here nor there in regards to this change, which is removing a feature that was relied on.

**Comment by tahonermann** [comment]
> I advise against using https://github.com/nextcloud/server/pull/36756 or https://github.com/nextcloud/server/pull/39702.

@kesselb, unfortunately, I believe Nextcloud offers no alternative at present. Per #2706, there is no ability to delegate calendar management to another user or group. Per [#26668](https://github.com/nextcloud/server/issues/26668), invitations aren't sent if a non-owning user edits a shared calendar.

The removal of notifications for edits to shared calendars effectively removed the only working method for group management of a calendar. Or do you perhaps know of some other way to configure a calendar so that it can 1) be edited by multiple users, and 2) have such edits be reflected in notifications? I can't find any other documented way to accomplish this. Use of a shared account is not an attractive option for several reasons.

**Comment by nunoleite** [comment]
The lack of functionalities centered on Projects and not only on Users is a problem for project managers. And we, as a user, have to invent some workarounds. If these workarounds are being limited then the all use of Nextcloud for these things is being limited.

Note to Nextcloud development... please start thinking in implementing something for Project Management... Centered in Projects... If we (Nextcloud community) want to be a ms teams and others alternative, we have to move that way. That's what i think. We have all the projects things spread across multiple users and multiple owners. This is no good. Stop implementing new features like crazy and work on workflows... These week another extension died "Approval".
Sorry this is off-topic, but it's a problem we have when using Nextcloud since the early days.

**Comment by ypatios** [comment]
We have also been negatively affected by this change. As a relatively large org with about 60 people, we regularly use several shared NC calendars and we are used to being able to edit the attendee list.
Please consider fixing the invitation issue instead of removing this sensible feature (sensible because if you have edit rights in a calendar you should be able to add/invite attendees).
Thanks for reading

**Comment by artforlife** [comment]
> The lack of functionalities centered on Projects and not only on Users is a problem for project managers. And we, as a user, have to invent some workarounds. If these workarounds are being limited then the all use of Nextcloud for these things is being limited.
> 

I too find the absence of this functionality highly inconvenient. I am managing a team and was sure that my assistant would be able to add people to events on a shared (mine) calendar. I was quite surprised to find out that this functionality was removed. It is quite inconvenient. We use Nextcloud because we care for our data and like the all-in-on-place approach. However, it is changes or discoveries like these that make life with Nextcloud quite frustrating. I truly hope that this gets reconsidered or fixed some other way.

**Comment by kesselb** [comment]
https://github.com/nextcloud/calendar/issues/5674#issuecomment-2271896247

**Comment by artforlife** [comment]
> [#5674 (comment)](https://github.com/nextcloud/calendar/issues/5674#issuecomment-2271896247)

Great news, thanks!

**Pull request merged**